cli/azd/cmd/env.go

```diff
@@ -514,39 +514,43 @@ func (eg *envGetValuesFlags) Bind(local *pflag.FlagSet, global *internal.GlobalC
 }
 
 type envGetValuesAction struct {
-	azdCtx    *azdcontext.AzdContext
-	console   input.Console
-	env       *environment.Environment
-	formatter output.Formatter
-	writer    io.Writer
-	flags     *envGetValuesFlags
+	azdCtx     *azdcontext.AzdContext
+	console    input.Console
+	envManager environment.Manager
+	formatter  output.Formatter
+	writer     io.Writer
+	flags      *envGetValuesFlags
 }
 
 func newEnvGetValuesAction(
 	azdCtx *azdcontext.AzdContext,
-	env *environment.Environment,
+	envManager environment.Manager,
 	console input.Console,
 	formatter output.Formatter,
 	writer io.Writer,
 	flags *envGetValuesFlags,
 ) actions.Action {
 	return &envGetValuesAction{
-		azdCtx:    azdCtx,
-		console:   console,
-		env:       env,
-		formatter: formatter,
-		writer:    writer,
-		flags:     flags,
+		azdCtx:     azdCtx,
+		console:    console,
+		envManager: envManager,
+		formatter:  formatter,
+		writer:     writer,
+		flags:      flags,
 	}
 }
 
 func (eg *envGetValuesAction) Run(ctx context.Context) (*actions.ActionResult, error) {
-	err := eg.formatter.Format(eg.env.Dotenv(), eg.writer, nil)
-	if err != nil {
-		return nil, err
+	env, err := eg.envManager.Get(ctx, "")
```

Comment:
I feel like we should be pulling the default environment name from the `azdcontext`, `azdContext.GetDefaultEnvironmentName()`  and passing that value to `envManager.Get(ctx, name)` instead of an empty string.

Reply:
I also feel like this same error might also exist on `azd env refresh` command when the environment hasn't been initialized.

Reply:
> I also feel like this same error might also exist on `azd env refresh` command when the environment hasn't been initialized.

@wbreza I don't get your second comment.. do you want me to implement this on the env refresh or are you saying that this error will be triggered when someone tries to run `azd env refresh`.

Reply:
> I feel like we should be pulling the default environment name from the `azdcontext`, `azdContext.GetDefaultEnvironmentName()` and passing that value to `envManager.Get(ctx, name)` instead of an empty string.

done ✅